### ISSUE TITLE
Accessibility: Evaluate and Improve Accessibility of mybinder.org

### ISSUE THREAD

**Issue by liliyao2022** [jupyterhub/outreachy#131]
### What page is this for?

https://mybinder.org/

### WAVE accessibility report

<img width="1403" alt="Screenshot 2022-11-02 at 5 03 25 pm" src="https://user-images.githubusercontent.com/103026397/199411084-a2f6e117-90ae-41d6-8a0b-7de17c6e307e.png">

After evaluating this page with the WAVE tool,  I found there are several errors that needed to be improved.

**8 Contrast Errors**  which means very low contrast between text and background colours.
There is 8 X Very low contrast.
Why It Matters
Adequate contrast of text is necessary for all users, especially users with low vision.

**3 Alerts** which means An event handler is present that may not be accessible.
There are  3 Alerts, which are:
1 X No page regions
1 X Skipped heading level
1 X Device dependent event handler
Why It Matters
The JavaScript events in use do not appear to be accessible to both mouse and keyboard users. To be fully accessible, critical JavaScript interaction should be device independent.

**11 Features** which means Image alternative text is present.
Why It Matters
The alternative text presents the content or function of an image to screen reader users or in other situations where images cannot be seen or are unavailable.

**7 Structural Elements** which means a first-level heading is present.
Why It Matters
Headings facilitate page navigation for users of assistive technologies. They also provide semantic and visual meaning and structure to the document. First-level headings should contain the most important heading(s) on the page (generally the document title).

**7 ARIA** which means an ARIA role, state, or property is present.
Why It Matters
ARIA provides enhanced semantics and accessibility for web content.

### The HTML element to be changed

```html
**8 Contrast Errors**
<a href="https://cloud.google.com/" style="color: rgb(51, 122, 183); background-color: rgb(235, 236, 237);">
Google Cloud
</a>

<a class="btn" style="width:fit-content;height:fit-content;padding:10px;background-color:#e66581;color:white;font-weight:bold;margin:-8px 0px;" onmouseover="this.style.backgroundColor='#d15b75'" onmouseout="this.style.backgroundColor='#e66581'" href="https://numfocus.salsalabs.org/donate-to-binder" target="_blank">
🤍 Donate to mybinder.org!
</a>

<button id="submit" class="btn-submit" type="submit">
launch
</button>

<span class="point point-orange">
1
</span>


<span class="point point-blue">
3
</span>


**3 Alerts** 
<head>
<base href="https://mybinder.org/">
<title>
Binder
</title>
<meta property="og:title" content="The Binder Project">
<meta property="og:image" content="https://mybinder.org/static/images/logo_social.png">
<meta property="og:description" content="Reproducible, sharable, open, interactive computing environments.">
<meta property="og:image:width" content="1334">
<meta property="og:image:height" content="700">
<meta property="og:image:alt" content="The Binder Project Logo">
<meta name="twitter:card" content="summary_large_image">
<meta id="base-url" data-url="/">
<meta id="badge-base-url" data-url="https://mybinder.org/">
<script type="text/javascript" async="" defer="" src="//wave.webaim.org/matomo/piwik.js"></script>
<script src="/static/dist/bundle.js?v=10a9e9f49b3cd457fb2aebb5c7e1436837a6b21a37b73dd4a5ee517765cbaa1f355074640fb6cfd3a45f0599e2345a5821111cc06bc2cab045a796756add031a"></script>
<link id="favicon" rel="shortcut icon" type="image/png" href="/static/favicon.ico?v=a3a57d0630d11611c499ba1adad64b1ecff1e9c3b4f2b2c1abffe4cd3ce642378418eddc6b6df9b7b799ba41364c60e7daac2bd65f45784c30ed58a6ed798c7e">
<link href="/static/dist/styles.css?v=cfcc3b6e823fb5767df94350ade3dcd4e8b665d40d2201523b6da5caea7b868c45f3c85b213d5159124a337229810fe140b91482254678f2631e5b1a4bea4a5f" rel="stylesheet">
</head>

2, Skipped heading level
<a class="btn" style="width: fit-content; height: fit-content; padding: 10px; background-color: rgb(230, 101, 129); color: white; font-weight: bold; margin: -8px 0px;" onmouseover="this.style.backgroundColor='#d15b75'" onmouseout="this.style.backgroundColor='#e66581'" href="https://numfocus.salsalabs.org/donate-to-binder" target="_blank">
🤍 Donate to mybinder.org!
</a>

3,Device dependent event handler, needs to add some JavaScript code, not this time.
```

### Your proposed HTML (or CSS), after change

```html
**8 Contrast Errors solutions: Increase the contrast between the foreground (text) colour and the background colour or enlarge the font size.** 

1, changed colour
<a href="https://cloud.google.com/" style="color: red; background-color: rgb(235, 236, 237);">
Google Cloud
</a>
2, changed colour
<a class="btn" style="width:fit-content;height:fit-content;padding:10px;background-color:#e66581;color:yellow;font-weight:bold;margin:-8px 0px;" onmouseover="this.style.backgroundColor='#d15b75'" onmouseout="this.style.backgroundColor='#e66581'" href="https://numfocus.salsalabs.org/donate-to-binder" target="_blank">
🤍 Donate to mybinder.org!
</a>
3, enlarged font-size
<button id="submit" style="font-size: 20px" class="btn-submit" type="submit">
launch
</button>
4, changed colour
<span style="color: rgb(42, 247, 125)";class="point point-orange">
1
</span>
5, changed colour
<span style="color: rgb(247, 213, 42) class="point point-blue">
3
</span>


**3 Alerts solutions** 

1,  the first alert needs to mark up different regions of the whole web page by adding the <header> element, which can be used to define such a region.

<header>
<base href="https://mybinder.org/">
<title>
Binder
</title>
<meta property="og:title" content="The Binder Project">
<meta property="og:image" content="https://mybinder.org/static/images/logo_social.png">
<meta property="og:description" content="Reproducible, sharable, open, interactive computing environments.">
<meta property="og:image:width" content="1334">
<meta property="og:image:height" content="700">
<meta property="og:image:alt" content="The Binder Project Logo">
<meta name="twitter:card" content="summary_large_image">
<meta id="base-url" data-url="/">
<meta id="badge-base-url" data-url="https://mybinder.org/">
<script type="text/javascript" async="" defer="" src="//wave.webaim.org/matomo/piwik.js"></script>
<script src="/static/dist/bundle.js?v=10a9e9f49b3cd457fb2aebb5c7e1436837a6b21a37b73dd4a5ee517765cbaa1f355074640fb6cfd3a45f0599e2345a5821111cc06bc2cab045a796756add031a"></script>
<link id="favicon" rel="shortcut icon" type="image/png" href="/static/favicon.ico?v=a3a57d0630d11611c499ba1adad64b1ecff1e9c3b4f2b2c1abffe4cd3ce642378418eddc6b6df9b7b799ba41364c60e7daac2bd65f45784c30ed58a6ed798c7e">
<link href="/static/dist/styles.css?v=cfcc3b6e823fb5767df94350ade3dcd4e8b665d40d2201523b6da5caea7b868c45f3c85b213d5159124a337229810fe140b91482254678f2631e5b1a4bea4a5f" rel="stylesheet">
</header>

2, Skipped heading level, add an <h4> element
<a class="btn" style="width: fit-content; height: fit-content; padding: 10px; background-color: rgb(230, 101, 129); color: white; font-weight: bold; margin: -8px 0px;" onmouseover="this.style.backgroundColor='#d15b75'" onmouseout="this.style.backgroundColor='#e66581'" href="https://numfocus.salsalabs.org/donate-to-binder" target="_blank"> <h4>🤍 Donate to mybinder.org!</h4>
</a>
```

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
